scripts/postinst

```diff
@@ -1,13 +1,11 @@
 #!/bin/sh
 
 set -e
+Python_SITELIB=$(python3 -c "from distutils import sysconfig;print(sysconfig.get_python_lib(plat_specific=False,standard_lib=False))")
```

Comment:
As of python 3.10 (in Ubuntu 22), `distutils` is deprecated. It will be removed in 3.12. Additionall pain: distutils must be installed before use!

I'm not 100% certain this suggestion works in non Debian derived environments, but it works in my Debain Python 3.11 environment as well as a mostly vanilla Ubuntu 18 environment. [Here is the documentation on Debian dist-packages.](https://wiki.debian.org/Python)
```suggestion
Python_SITELIB=$(python3 -c "import sysconfig; print(sysconfig.get_path('stdlib'))")/dist-packages
```

If this is not the solution, we should find a better solution than `distutils`.

Reply:
I couldn't get that to work in Ubuntu nor Debian
```
# ls $(python3 -c "import sysconfig; print(sysconfig.get_path('stdlib'))")/dist-packages
ls: cannot access '/usr/lib/python3.8/dist-packages': No such file or directory
```

Reply:
I think you want `platlib`

```
# python3 -c "import sysconfig;  print(sysconfig.get_path('platlib'))"
/usr/lib/python3.9/site-packages
```

Reply:
Currently, it is implemented with `python3 -c "import sysconfig; print(sysconfig.get_path('purelib'))"` as described in https://cmake.org/cmake/help/latest/module/FindPython3.html

Reply:
According to that link, dist-packages seem to be the correct path. I wonder if we need to `mkdir -p` to ensure it exists. @spoonincode This worked for me in Debian 11/12 and Ubuntu 18.

Reply:
`purelib` seems to be the correct answer. :-) Approved.

Reply:
Are we sure we should be installing into `site-packages`?

```dist-packages instead of site-packages. Third party Python software installed from Debian packages goes into dist-packages, not site-packages. This is to reduce conflict between the system Python, and any from-source Python build you might install manually.```
https://wiki.debian.org/Python